### PR TITLE
feat(fx-2614): support multiple colors in filters

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [FX-2614](https://artsyproduct.atlassian.net/browse/FX-2614)

Let's take a moment to appreciate that this error-prone commit — https://github.com/artsy/force/commit/e7ba68752b3a1dfba5609eff7aefb149bfc860e8 — shouldn't even exist. Let's convert the input to an input object and never do this again 😅 

Draft because it's Friday at 5:45. Will open on Monday 🦢

### CODE REVIEW

File: src/v2/DevTools/setupTestWrapper.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ type SetupTestWrapper<T extends OperationType> = {
       return <ShowMetaFragmentContainer show={props.show} />
     },
     query: graphql`
-      query OverviewRoute_Test_Query($showID: String!) {
+      query ExampleOverviewRoute_Test_Query($showID: String!) {
```
Comment (dzucconi) [inline src/v2/DevTools/setupTestWrapper.tsx:33]: lol it complained about this one too @damassi

File: src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx [hunk 2]
```diff
@@ -67,6 +67,7 @@ export const ArtistArtworkFilterRefetchContainer = createRefetchContainer(
           atAuction: { type: "Boolean" }
           attributionClass: { type: "[String]" }
           color: { type: "String" }
+          colors: { type: "[String]" }
```
Comment (mzikherman) [inline src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx:70]: Should we be removing the `color` arg? (now unused)

Comment (dzucconi) [reply src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx:70]: Might as well

**Comment by dzucconi** [comment]
Should be good to go @mzikherman

**Review by dblandin — APPROVED** [review]
LGTM 👍

**Pull request merged**